packages/vscode-docker-registries/src/clients/GenericRegistryV2/GenericRegistryV2WizardPromptStep.ts

```diff
@@ -0,0 +1,20 @@
+import { Uri } from 'vscode';
+import { RegistryWizardPromptStep } from '../../wizard/RegistryWizardPromptStep';
+import { showInputBox } from '../../wizard/showInputBox';
+import { RegistryWizardContext } from '../../wizard/RegistryWizardContext';
+
+export interface GenericRegistryV2WizardContext extends RegistryWizardContext {
+    readonly registryPrompt: string;
+    registryUri?: Uri;
+}
+
+export class GenericRegistryV2WizardPromptStep<T extends GenericRegistryV2WizardContext> extends RegistryWizardPromptStep<T> {
+    public async prompt(wizardContext: T): Promise<void> {
+        const url = await showInputBox({ isSecretStep: false, prompt: wizardContext.registryPrompt || '' });
```

Comment:
We should add a validation method to ensure that the input parses

Reply:
yeah we should